public/locales/fr/expectations.json

```diff
@@ -29,7 +29,17 @@
       "item-2": "pour les affaires officielles du gouvernement (passeports spéciaux ou diplomatiques)"
     }
   },
-  "description-privacy": "Avez-vous lu et accepté les conditions d'utilisation et de confidentialité?",
+  "description-privacy": {
+    "1": "Les renseignements fournis sont recueillis en vertu de l'article 5.1(1)(a)(i) de la Loi sur le ministère de l'Emploi et du Développement social et/ou de l'article 12(e) du Décret sur les passeports canadiens dans le but de vous fournir l'état de votre demande de passeport. Si vous refusez de fournir des renseignements, vous ne serez pas en mesure de consulter l'état de votre demande de passeport.",
+    "2": "De plus, nous ne recueillons votre adresse courriel que si vous demandez à Emploi et Développement social Canada (EDSC)C d'envoyer votre numéro de dossier de demande de passeport par courriel. Sans votre adresse courriel, EDSC ne peut pas envoyer votre numéro de demande de passeport par voie électronique.",
+    "3": "Your personal information may be shared within Service Canada and with Immigration, Refugees and Citizenship Canada to improve services through evaluations, reports and research. This will never result in an administrative decision being made about you.",
+    "4": "EDSC exécute cette application de vérification d'état en utilisant les services du fournisseur tiers Microsoft Corporation, qui peuvent être assujettis à des lois étrangères. Pour plus d'information, consultez la <2>Politique de confidentialité de Microsoft</2>.",
+    "4-link": "https://www.microsoft.com/fr-ca/trust-center/privacy",
+    "5": "Vos renseignements personnels sont administrés conformément à la Loi sur le ministère de l'Emploi et du Développement social, la Loi sur la protection des renseignements personnels et les autres lois applicables. Vous avez le droit d'accéder à vos renseignements personnels et de les protéger, tel que décrit dans le fichier de renseignements personnels - Programme de passeport (EDSC PPU 708). Les instructions pour obtenir ces informations sont décrites dans la publication gouvernementale intitulée <2>Renseignements sur les programmes et les fonds de renseignements</2>.",
+    "5-link": "https://www.canada.ca/fr/emploi-developpement-social/ministere/transparence/aai/rapports/infosource.html",
+    "6": "Vous avez le droit de déposer une plainte auprès du Commissariat à la protection de la vie privée du Canada au sujet du traitement de vos renseignements personnels par EDSC: <2>https://www.priv.gc.ca/fr/signaler-un-probleme</2>.",
```

Comment:
Add `\u00a0` before the colon in `EDSC:`

Reply:
https://github.com/DTS-STN/passport-status/pull/240#discussion_r1071492001